lib/IonKey.ts

```diff
@@ -1,12 +1,18 @@
 import * as Ed25519 from '@noble/ed25519';
 import * as Secp256k1 from '@noble/secp256k1';
-import InputValidator from './InputValidator';
-import IonPublicKeyModel from './models/IonPublicKeyModel';
-import IonPublicKeyPurpose from './enums/IonPublicKeyPurpose';
-import JwkEd25519 from './models/JwkEd25519';
-import JwkEs256k from './models/JwkEs256k';
+import InputValidator from './InputValidator.js';
+import IonPublicKeyModel from './models/IonPublicKeyModel.js';
+import IonPublicKeyPurpose from './enums/IonPublicKeyPurpose.js';
+import JwkEd25519 from './models/JwkEd25519.js';
+import JwkEs256k from './models/JwkEs256k.js';
 import { base64url } from 'multiformats/bases/base64';
 
+// NOTE: @noble/secp256k1 requires globalThis.crypto polyfill for node.js <=18: https://github.com/paulmillr/noble-secp256k1/blob/main/README.md#usage
+// Remove when we move off of node.js v18 to v20, earliest possible time would be Oct 2023: https://github.com/nodejs/release#release-schedule
+import { webcrypto } from 'node:crypto';
```

Comment:
Not all projects are going to have webcrypto or use node though, right?

Reply:
(same as my below comment)

Reply:
Same as the other.